### PR TITLE
[v6r15] Making pylint happier

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: AccountingSystem/private/MainReporter.py [hunk 1]
```diff
@@ -1,20 +1,19 @@
-try:
-  import hashlib
-  md5 = hashlib
-except:
-  import md5
-import re  
+""" Accounting reporter
+"""
+
+import hashlib as md5
```
Comment (chaen) [inline AccountingSystem/private/MainReporter.py]: can't we just import hashlib as hashlib now ? :D

File: Core/Workflow/Module.py [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-__RCSID__ = "$Id:$"
-
 import copy
 import os
 
 #try: # this part to import as part of the DIRAC framework
 from DIRAC.Core.Workflow.Parameter import *
 
+__RCSID__ = "$Id:$"
```
Comment (atsareg) [inline Core/Workflow/Module.py]: $Id$

File: Core/Workflow/Step.py [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 import time
-import types
 import traceback
 import sys
 
 from DIRAC.Core.Workflow.Parameter import *
 from DIRAC.Core.Workflow.Module import *
 from DIRAC import S_OK, S_ERROR
 
+__RCSID__ = "$Id:$"
```
Comment (atsareg) [inline Core/Workflow/Step.py]: Same

**Comment by atsareg** [comment]
Review OK, with minor comments

**Pull request merged**